sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -139,3 +139,8 @@
 .v-expansion-panel-title .v-icon {
   color: $brand-blue;
 }
+
+// Changed the style to match with Figma.
+.v-btn--variant-outlined {
+  background-color: white !important;
```

Comment:
is it not white by default?

Reply:
Nope, it is transparent which makes it gray.